### PR TITLE
Bump `package_info_plus` 

### LINKED ISSUE #1944: Upgrade Package_info_plus to version 6
### Description

sentry_flutter 7.18.0 depends on package_info_plus >=1.0.0 <=5.0.1

### PR DESCRIPTION
**Pull request by denrase**
#skip-changelog

## :scroll: Description

expand package_ino_plus range to 6.0.0

## :bulb: Motivation and Context

Closes #1944

## :green_heart: How did you test it?

CI

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] No new PII added or SDK only sends newly added PII if `sendDefaultPii` is enabled
- [ ] I updated the docs if needed
- [ ] All tests passing
- [x] No breaking changes

### CODE REVIEW

File: flutter/pubspec.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ dependencies:
   flutter_web_plugins:
     sdk: flutter
   sentry: 7.18.0
-  package_info_plus: '>=1.0.0 <=5.0.1'
+  package_info_plus: '>=1.0.0 <=6.0.0'
```
Comment (ueman) [inline flutter/pubspec.yaml]: Shouldn't it be the following since there's already v6?
```suggestion
  package_info_plus: '>=1.0.0 <=7.0.0'
```

Comment (denrase) [reply flutter/pubspec.yaml]: 7.0.0 does not exist and might contain breaking changes once released. We could probably go with `<7.0.0` if we assume 6.x version will not have breaking changes.

Comment (ueman) [reply flutter/pubspec.yaml]: Oh yeah, you're right. `<7.0.0` is what I meant

**Review by kahest — APPROVED** [review]

**Comment by aggeloskoutanis** [comment]
Despite that 7.18 version is live on pub.dev, somehow I can still see in pubspec.yaml `package_info_plus: '>=1.0.0 <=5.0.1'`

**Comment by kahest** [comment]
@aggeloskoutanis that's correct, we didn't release a new SDK version with that change yet!

**Comment by aggeloskoutanis** [comment]
Any ETA on when this will be available on pub.dev?

**Comment by kahest** [comment]
We're planning on early next week cc @stefanosiano @denrase

**Comment by rekire** [comment]
Not exactly this week right? Might be next week? 😁

**Pull request merged**